src/styles/index.scss

```diff
@@ -1,3 +1,136 @@
+$background-color: #fff;
+$font-color: #060b35;
+$details-color: #616070;
+$theme-color: #00acdc;
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  margin: 0;
+}
+
 body {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
```

Comment:
You should put color value to a variable, and name it e.g. `card-border-color`, so you could easily find all of your colors later.